### DISCUSSION TITLE
Features in Lookup fields/table (Collections Relationship)

### DISCUSSION THREAD

**Discussion by suhaimim** [directus/directus#9937] [Archive]
1. May I know what is the best way to filter (grouping) the lookup fields (relationship)?

STATES is the LOOKUP TABLE for this District collections.
![image](https://user-images.githubusercontent.com/35832684/142721115-2a3a2557-692d-4e47-8ff2-fddb8571891d.png)

Once the filter tries to filter (group by states), none of the list is matched and shows nothing.
![image](https://user-images.githubusercontent.com/35832684/142721172-9aef405f-637a-4dee-8026-ca481b35586d.png)

2. May I know how to show/display the COUNTRY's name in the sale list/table, since LOOKUP TABLE for STATES is also has relation to COUNTRY collection.

![image](https://user-images.githubusercontent.com/35832684/142721223-abbd5f49-e512-48ae-9f8b-9efcd452ed83.png)

**Comment by rijkvanzanten** [comment]
The search bar will search through any of the text fields in the current table. You can use the filter button next to it to search in specific related fields, like "states.states_name"

**Reply by suhaimim** [reply]
https://github.com/directus/directus/discussions/9967

**Comment by suhaimim** [comment]
May I know how to apply this into the filtering box?

![image](https://user-images.githubusercontent.com/35832684/142977313-39747e76-59c4-4165-a758-adda19e59962.png)

![image](https://user-images.githubusercontent.com/35832684/142977248-eab0f487-3e2f-406d-a3e4-d02f4e3a1107.png)

![image](https://user-images.githubusercontent.com/35832684/142977356-52e87c82-cfa9-4c3d-a86f-2e12e4d864fa.png)

![image](https://user-images.githubusercontent.com/35832684/142977514-e5813148-cbc9-4f2c-8536-0a7f4b9ff9ab.png)
